tests/unit_tests/snapshots/test_libres_facade/test_load_gen_kw_not_sorted/gen_kw_unsorted

```diff
@@ -1,11 +1,11 @@
-Realization,PARAM_2:MY_KEYWORD,PARAM_1:MY_KEYWORD
```

Comment:
I had to update the whole snapshot because I changed the parameter from `NORMAL` -> `LOGUNIF`. I created the snapshot on version 4.5 to make sure nothing changed.